components/TransLine.tsx

```diff
@@ -15,7 +15,11 @@ export interface TransLineProps extends TransLineContent {
 function resolveUrl(link: I18nString, userArrivedFromUioMobile: boolean) {
   // Special case for UIO homepage links.
   if (link === 'uio-home') {
-    const uioHomeLink = userArrivedFromUioMobile ? getUrl('uio-home-url-mobile') : getUrl('uio-home-url-desktop')
+    // Optional environment-specific links back to the UIO landing page, used by EDD testing
+    const uioLandingUrl = process.env.URL_UIO_LANDING ?? getUrl('uio-home-url-desktop')
+    const uioMobileLandingUrl = process.env.URL_UIOMOBILE_LANDING ?? getUrl('uio-home-url-mobile')
+
+    const uioHomeLink = userArrivedFromUioMobile ? uioMobileLandingUrl : uioLandingUrl
```

Comment:
Nit: Not necessary for launch, but I think this is enough logic to put it somewhere else (like in `getUrl.ts`) rather than in the component.

Reply:
Oh yeah, I should have put it there. Moved!